src/actions/reports/cost.py

```diff
@@ -146,20 +181,51 @@ def visit_ScanNode(self, node):
         if self.depth == 0:
             self.__exit()
 
+    @staticmethod
+    def set_seq_scan_node_features(feat, node):
+        feat.has_local_filter = int(node.get_local_filter() != None)
+        feat.has_table_filter_seqscan |= int(node.get_remote_filter() != None)
+
+    @staticmethod
+    def set_index_scan_node_features(feat, node):
+        feat.has_local_filter = int(node.get_local_filter() != None)
```

Comment:
` != None` should be ` is not None`

Reply:
`!= None` works fine, too. But I see, because [`None` is the singleton instance](https://docs.python.org/3/library/constants.html#None), [identity check makes much more senses than equality](https://docs.python.org/3/reference/expressions.html#is-not).  Thanks!